doc/api/crypto.md

```diff
@@ -3622,6 +3622,7 @@ See the [list of SSL OP Flags][] for details.
 [RFC 5208]: https://www.rfc-editor.org/rfc/rfc5208.txt
 [encoding]: buffer.html#buffer_buffers_and_character_encodings
 [initialization vector]: https://en.wikipedia.org/wiki/Initialization_vector
+[safe integers]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
 [scrypt]: https://en.wikipedia.org/wiki/Scrypt
 [stream-writable-write]: stream.html#stream_writable_write_chunk_encoding_callback
```

Comment:
Optional nit: While we're in here, maybe ASCII-sort these last three links? `stream` should come before `stream-writable-write` and `list of SSL OP Flags` belongs above the `safe integers` link you're adding here.

Reply:
I copied those changes from your closed PR and added you as a co-author, hope that's okay with you.